### DISCUSSION TITLE
May 14 Writers Workshop

### DISCUSSION THREAD

**Discussion by tonyfast** [Quansight/writers-workshop#7] [New class]
# the goal of the workshop

understand what we need to write? how does reading effect writing?

publish together, write together

## teacher's preferred ground rules

* How to use the raise handle feature? Keeping the professor on track..
* I encourage y'all to interrupt me when I'm talking or demo'ing. 
  Sometimes I get distracted by my own special interests and this class if for you all.
* To help take notes, please respond to this discussion item and continue editing your one reply during class. i can see the changes on the discussion board as they happen.

## [call to action] :+1: :-1:  :smile: :confused:  :rocket:  :heart:  :eyes: :tada: attendance

react to this discussion so we can take attendance.

![image](https://user-images.githubusercontent.com/4236275/117461813-fd163180-af1b-11eb-901b-bfdee65d8bdc.png)

### Q1 how's your energy

* :+1: i feel energetic from this week
* :-1: i feel exhausted from this week

### Q3 how do you want to participate

* :heart:   i'm happy to be at the writers workshop this week
* :eyes: i want to just watch and listen
* :tada: i am okay being called on for discussions

## agenda

0. [:00-:07] chit chat
1. [:07-:12] Recap and introduce new folks

2. [:12-:40] discuss what we need to write
3. [:40-:50] talk about individual projects we can work on

4. [:50-:60] wrapup

    feedback and improvements

**Comment by isabela-pf** [comment]
Notes:
### Intro
- Just write things! Don't let the format or any decisions stop you! There's always people and docs that can help you with production when you get there.
- Quick recap of past meetings to get new people up to date.
- If you haven't written a github profile readme, give that a go!

### What do you need to write?
Ideas included
- Time. Long blocks or designated, consistent time.
- Paper and pencil. The space to think nonlinearly. 
- To get over fear of the ["shitty first draft"](https://wrd.as.uky.edu/sites/default/files/1-Shitty%20First%20Drafts.pdf). Especially if you are in a collaborative space where you need review or need to write together.
- Being comfortable
- Making a space for enjoyment.
- Emacs
- Electronic music

### How posts get added to this repo
- Tony shares his process of pulling in blog content
- [doit](https://pydoit.org/contents.html)
- Most blogging platforms use yaml front matter, so Tony encourages using that syntax now

### What's next
- Just write something. This could be continuing an existing work, starting a new work, or going back to the github profiles. 
- Tony says he wants to fix images and LaTeX support in the docs.
